Domain/Dto/BetRequest/CreateBetRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Domain.Dto.BetQuote;
+using Domain.Dto.Bets;
+using Domain.Entities;
+
+namespace Domain.Dto.BetRequest
+{
+    public class CreateBetRequest
+    {
+        public CreateBetsDto Bet { get; set; }
+        public CreateBetQuotesDto BetQuote { get; set; }
```

Comment:
we could have all the properties in here, without the need of having two distinct objects